packages/features/ee/teams/components/TeamList.tsx

```diff
@@ -71,10 +68,14 @@ export default function TeamList(props: Props) {
                   <Card
                     icon={<Icon name="building" className="h-5 w-5 text-red-700" />}
                     variant="basic"
-                    title={t("You have a lot of teams")}
-                    description={t(
-                      "Consider consolidating your teams in an organisation, unify billing, admin tools and analytics."
-                    )}
+                    title={props.teams.length === 1 ? t("you_have_one_team") : t("You have a lot of teams")}
+                    description={
+                      props.teams.length === 1
+                        ? t("consider_consolidating_one_team_org")
+                        : t(
```

Comment:
extract this? "Consider consolidating your teams in an organisation, unify billing, admin tools and analytics."